src/app/components/translate-attributes/translate-attributes.component.html

```diff
@@ -0,0 +1,98 @@
+<div class="modal-body">
+  <div class="d-flex justify-content-between align-items-center">
+    <h5>Translation Attributes</h5>
+    <!-- Add button -->
+    <button
+      class="btn btn-secondary add-new-translation-attribute"
+      (click)="createAttribute()"
+    >
+      <i class="fa fa-file"></i> Add new Translation Attribute
+    </button>
+  </div>
+
+  <!-- Loop through items and render -->
+  <div class="translation-attributes" *ngIf="resource">
+    <div
+      *ngIf="!resource['translated-attributes'].length"
+      class="no-attributes"
+    >
+      <p>They're are currently no translated attributes</p>
+    </div>
+
+    <div
+      class="row md-2"
+      *ngFor="let attr of resource['translated-attributes']"
```

Comment:
can we sort the `translated-attributes` based on their key? I got this unsorted order when opening the modal.

![Screen Shot 2022-12-05 at 10 48 27 AM](https://user-images.githubusercontent.com/47818/205681240-30e28c41-2c49-4bc9-8877-bd33caf9d7b5.png)

Reply:
because it would probably re-order things as you are updating attributes, we may just want to sort the attributes when we initially load them from the API, then leave them alone as the admin is modifying them.

Reply:
I've added a sort on the init load of the attributes that sorts them alphabetically on their `key`.